helix-term/src/ui/picker.rs

```diff
@@ -917,6 +843,8 @@ impl<T: Item + 'static + Send + Sync> Component for Picker<T> {
             }
         }
 
+        helix_event::send_blocking(&self.preview_highlight_handler, ());
```

Comment:
I am not too sure how we handle this currently but this may actually not be sufficient. If the matcher is still running in the background we may stream in new files. I didn't realize this when working on nucleo. What do you think of sending an event every time we render and don't have a preview instead?

Reply:
we only did this here in the past because we only had access to the compositor here to reset the idle timeout